static/app/components/events/eventStatisticalDetector/breakpointChart.tsx

```diff
@@ -29,6 +29,52 @@ type EventBreakpointChartProps = {
   event: Event;
 };
 
+type BreakpointChartData = {
+  event: Event;
+  eventStatsReponse:
+    | {
+        data: EventsStatsData;
+        meta: MetaType;
+      }
```

Comment:
I'm a noob at js/ts, what does the first `|` do?

Reply:
It's type union here, basically this object can be an object that has `data` and `metadata` in it or it can be undefined based on what backend returns.